task_spec/spec/acceptance/init_spec.rb

```diff
@@ -37,8 +37,10 @@ def target_platform
                          '6.18.0'
                        when %r{osx-10.15}
                          '6.15.0'
+                       when %r{osx-11}
+                         '6.23.0'
                        else
-                         '6.17.0'
+                         '6.23.0'
```

Comment:
This is making the task spec fail on all the other OSes since we didn't release 6.23.0 yet